plugins/ReactNativeStyle/BoxInspector.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var React = require('react');
+
+type Props = {
+  style: Object,
+  left: number,
+  top: number,
+  width: number,
+  height: number,
+  margin: Object,
+  padding: Object,
+};
+
+type DefaultProps = {};
+
+type BoxProps = {
+  title: string,
+  box: {
+    top: number,
+    left: number,
+    right: number,
+    bottom: number,
+  },
+  children: React$Element,
+};
+
+var Box = (props: BoxProps) => {
+  var box = props.box;
```

Comment:
It's kinda weird to have a `Box` that takes a prop called `box`. It seems like there is a name for a domain concept here that wants to be found. Maybe creating a type for `{ top: number, left: number, right: number, bottom: number }` will help us find a clearer name to use for this prop.

Alternatively, perhaps `top`, `left`, `right`, and `bottom` could be made top-level props on the element itself:
```js
<Box top={4} left={5} right={6} bottom={7} />
```